### PR TITLE
Feature/us1248 render kids club application

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: crossroads.net/app/routes.js [hunk 1]
```diff
@@ -45,6 +46,7 @@
         registerType("contentRouteType", /^\/.*/);
         registerType("signupRouteType", /\/sign-up\/.*$/);
         registerType("volunteerRouteType", /\/volunteer-sign-up\/.*$/);
+        //registerType("volunteerApplicationRouteType", /\/volunteer-application\/.*$/);
```
Comment (silbermm) [inline crossroads.net/app/routes.js]: Commented Code

File: crossroads.net/app/volunteer_application/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+var app = require("angular").module("crossroads");
+require("./volunteer_application_form.html");
+require('../profile/personal/profile_personal.html');
+require('../profile');
+// require('./volunteer_application.config.js');
+
+app.controller("VolunteerApplicationController", require("./volunteer.application.controller"));
+app.factory("Opportunity", ["$resource", "Session", require('../services/opportunity_service')]);
+// app.factory("Opportunity", ["$resource", "Session", require('./opportunity_service')]);
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/index.js]: Commented code

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+      <span ng-show="false">Please provide a valid value.</span>
+    </div>
+    <div class="form-group col-sm-6">
+      <label class="control-label required" for="referenceOccupation-3">Occupation</label>
+      <input type="text" class="form-control" id="referenceOccupation-3" placeholder="Occupation">
+      <span ng-show="false">Please provide a valid value.</span>
+    </div>
+    <p class="help-text">Okay, here's some formal stuff. We don't really talk like this, but we couldn't figure out how to say all this in kid-friendly language. So put your glasses on, read it and agree to it, please The information contained in this application
+      is true. I authorize any reference or church listed here to give you any information (including opinions) that they may have regarding my character and fitness for working with kids. By agreeing to this form, I also consent to a complete
+      criminal background check now an din the future. I understand my involvement with Crossroads Kids' Club is influenced by the information provided by all these sources. I release from liability for damages any person or organizations identified
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: `now and in the future`

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+      by me in this application, and waive my right to inspect any written material from contact with them or from the criminal background check. I have carefully read the forgoing release and know the contents thereof and I agree to this release
+      as my own free act. This is a legally binding agreement which I have read and understand.</p>
+    <div class="form-group col-sm-6">
+      <label class="block required">Agree</label>
+      <label class="radio-inline">
+        <input type="radio" value="signature" name="gender">Yes
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: `name="gender"`?

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html [hunk 5]
```diff
@@ -0,0 +1,572 @@
+<form class="ng-pristine ng-valid-date ng-invalid ng-invalid-required ng-valid-time ng-valid-date-disabled ng-valid-maxlength" _lpchecked="1">
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: these classes are added dynamically by angular... did you copy and paste from chrome dev tools?

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html [hunk 6]
```diff
@@ -0,0 +1,572 @@
...
+  <fieldset class="push-bottom">
+    <legend>General Info</legend>
+    <div class="row">
+
+      <div class="form-group col-sm-6">
+        <label for="firstname" class="control-label required">First Name</label>
+        <input type="text" class="form-control" id="firstName" placeholder="Enter first name" ng-value="volunteer.person.firstName" disabled="true">
+        <span ng-show="false">Please provide a valid value.</span>
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: `ng-show=false` will never show this text. Is this intended?

File: crossroads.net/app/volunteer_application/opportunity_service.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+module.exports = function($resource, Session) {
+    return $resource(__API_ENDPOINT__ + "api/opportunity/:opportunityId");
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/opportunity_service.js]: to stay consistent, we should name this file opportunity.service.js 

I thought this already existed though

File: crossroads.net/app/volunteer_application/volunteer.application.controller.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
+"use strict";
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/volunteer.application.controller.js]: Name the file volunteerApplication.controller.js

File: crossroads.net/app/volunteer_application/volunteer.application.controller.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+      } else {
+        vm.showError = true;
+      }
+    }
+
+    function opportunityResponse() {
+      Opportunity.GetResponse.get({
+          id: vm.pageInfo.opportunity,
+          contactId: vm.contactId
+        }).$promise
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/volunteer.application.controller.js]: you don't need to explicitly call `$promise` here... the second argument to the function is a function that happens when the response is successful.

```javascript
Opportunity.GetResponse.get({},function(success){
   // the call was successful
}, function(error){
  // the call resulted in an error
});

```

File: crossroads.net/app/volunteer_application/volunteer_application_form.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
+<div class="col-sm-10 col-sm-offset-1">
```
Comment (silbermm) [inline crossroads.net/app/volunteer_application/volunteer_application_form.html]: name this file volunteerApplicationForm.template.html to be consistent

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html (hunk 6 above)
Comment (IngageMaddoxT) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: those are place holders for future validation messages.

**Comment by IngageMaddoxT** [comment]
@silbermm responded to all your comments.  Also added the messages from CMS.

File: crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html (hunk 6 above)
Comment (silbermm) [inline crossroads.net/app/volunteer_application/kc_adult_application/kidsClubAdultApplication.template.html]: :+1:

**Pull request merged**